package.json

```diff
@@ -7,6 +7,7 @@
     "node": "^12.18"
   },
   "scripts": {
+    "axe:ci": "axe http://localhost:3000, http://localhost:3000/activity-reports, http://localhost:3000/admin --dir ./reports/",
```

Comment:
Is there a way to define these URLs in a separate file or something like that? Thinking in the future when we have more this line will get hard to read.

Reply:
Ooo, I like that idea.  At very least we should parameterize it. I'll play around with that.